widget/ui/src/components/QuoteExpanded/QuoteExpanded.Skeleton.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import { ChainToken } from '../ChainToken';
+import { Skeleton } from '../Skeleton';
+
+import {
+  FlexCenter,
+  lastStepStyle,
+  LoadingStep,
+  OutputLoading,
+  RightSection,
+  RouteContainer,
+  Separator,
+  VerticalLine,
+} from './QuoteExpanded.styles';
+
+const ITEM_SKELETON_COUNT = 3;
+
+export function QuoteSkeletonExpanded() {
+  return (
+    <RouteContainer selected={false}>
```

Comment:
I think it is better to use the same RouteContainer and only change the content of it. This way the container will not rerender when switching the loading value. Also you can use other components like RouteHeader and Steps to prevent duplicate code and also keeping the size of elements consistent on loading switch.  And maybe also you can create component just for new utilizations like SkeletonHeader and use them in the main component.